### PR TITLE
Unit test fixes

### PR DESCRIPTION
**Pull request by ScottVerbeek**
Fixes a series of unit tests

### CODE REVIEW

File: src/loader/moodle_curl_lrs.php [hunk 1]
```diff
@@ -51,29 +51,32 @@ function load(array $config, array $events) {
 
         $url = utils\correct_endpoint($endpoint).'/statements';
         $auth = base64_encode($username.':'.$password);
-		
-		//don't send empty statement arrays GVM 25-01-21
+
+        //don't send empty statement arrays GVM 25-01-21
+        // OS-1095: Raise a exception since it should be logged as a failed event.
         if(count($statements) > 0) {
```
Comment (rhell4) [inline src/loader/moodle_curl_lrs.php]: This if statement needs to be changed, right now this is checking the array isn't empty and then throwing an exception saying it is empty.

File: tests/enchancement_jisc_skeleton.php [hunk 2]
```diff
@@ -74,11 +74,35 @@ protected function setUp(): void {
...
+        }
+    }
+
+    /**
+     * Remove anything done in the setup.
+     *
+     * @return void
+     */
+    protected function tearDown(): void
+    {
```
Comment (rhell4) [inline tests/enchancement_jisc_skeleton.php]: Minor thing, Moodle open brackets should be on the same line not a new line.

File: tests/xapi_test_case.php [hunk 3]
```diff
@@ -41,6 +41,33 @@ abstract class xapi_test_case extends \advanced_testcase {
      */
     abstract protected function get_test_dir();
 
+    /**
+     * Create a PHPUNIT_XAPI_TESTCASE constant so that the fix for WR322566 does not get applied in this test.
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
```
Comment (rhell4) [inline tests/xapi_test_case.php]: Open brackets should be on the same line.

File: tests/xapi_test_case.php [hunk 4]
```diff
@@ -41,6 +41,33 @@ abstract class xapi_test_case extends \advanced_testcase {
...
+        }
+    }
+
+    /**
+     * Remove anything done in the setup.
+     *
+     * @return void
+     */
+    protected function tearDown(): void
+    {
```
Comment (rhell4) [inline tests/xapi_test_case.php]: Open brackets should be on the same line.

**Review by rhell4 — CHANGES_REQUESTED** [review]

File: src/loader/moodle_curl_lrs.php (hunk 1 above)
Comment (ScottVerbeek) [reply src/loader/moodle_curl_lrs.php]: Very good catch. Resolved this.

**Pull request merged**